cli.js

```diff
@@ -17,26 +19,73 @@ var usage = `Usage
 
   cabal --db /path/to/db
 
+  OR
+
+  cabal youraliashere
+
   Options:
 
-    --nick    Your nickname.
-    --seeder  Start a headless seeder for the specified cabal key
+    --nick <name>         Use <name> as nick for this session
+    --seeder              Start a headless seeder for the specified cabal key
+    --new-alias <name>    Add an alias to your config. Must be used with --key
+    --config-nick <name>  Set <name> as your default nick from now on
 
 Work in progress! Learn more at github.com/cabal-club
 `
 
-if (args.key) {
-  args.key = args.key.replace('dat://', '').replace(/\//g, '')
-  args.db = rootdir + args.key
+function saveConfig(cfg) {
+  fs.writeFileSync(configPath, JSON.stringify(config, null, 2))
+}
+
+var config
+if (fs.existsSync(configPath)) {
+  // load config
+  try {
+    config = JSON.parse(fs.readFileSync(configPath))
+  } catch (SyntaxError) {
+    process.stderr.write("Something is wrong with your config.\n")
+  }
+} else {
+  // generate default config
+  config = {
+    keyAliases : {}
+  }
+  saveConfig(config)
+}
+
+if (args['new-alias']) {
+  if (args.key) {
+    config.keyAliases[args['new-alias']] = args.key
+    saveConfig(config)
+    process.stdout.write("alias " + args['new-alias'] + " saved.\n")
+    process.exit(0)
+  } else {
+    process.stderr.write(usage)
+    process.exit(1)
+  }
+}
+
+if (args['config-nick']) {
+  config.nick = args['config-nick']
+  saveConfig(config)
+  process.stdout.write("nick " + args['config-nick'] + " saved.\n")
+  process.exit(0)
```

Comment:
can't we continue here (open cabal)?

Reply:
Sure, just gotta make sure the user doesn't get the help text if there was no --key or --db supplied when they use --config-nick